{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/views/private.py

```diff
@@ -2,11 +2,17 @@
 
 import flask
 from keg.web import BaseView
-from keg_auth import CrudView, requires_user, requires_permissions
-
-from {{cookiecutter.project_namespace}} import forms
-from {{cookiecutter.project_namespace}} import grids
-from {{cookiecutter.project_namespace}}.model import entities
+from keg_auth import (
```

Comment:
just `import keg_auth` would be fine by me.

Reply:
Unless you feel strongly about this, I would rather not have to do `keg_auth.CrudBase` everywhere...